examples/http-rust-router-macro/src/lib.rs

```diff
@@ -1,18 +1,16 @@
+#![allow(dead_code, unused_imports)]
 use spin_sdk::{
-    http::{Params, Request, Response},
+    http::{IntoResponse, Params, Request, Response},
     http_component, http_router,
 };
 
 #[http_component]
-fn handle_route(req: Request) -> anyhow::Result<Response> {
+fn handle_route(req: Request) -> impl IntoResponse {
     let router = http_router! {
         GET "/hello/:planet" => api::hello_planet,
-        _   "/*"             => |_req, params| {
+        _   "/*"             => |_req: Request, params| {
             let capture = params.wildcard().unwrap_or_default();
-            Ok(http::Response::builder()
-                .status(http::StatusCode::OK)
-                .body(Some(capture.to_string().into()))
-                .unwrap())
+            Response::new(200, capture.to_string())
```

Comment:
I'm not seeing the body as output when triggering this component.

Reply:
Hmmm... It seems to be working for me:

```
$ curl -i localhost:3000/wow       
HTTP/1.1 200 OK
content-length: 3
date: Wed, 18 Oct 2023 07:03:05 GMT

wow
```